Revisione

gioxx opened the pull request:
- Ho cercato di dare un ordine ai progetti Mozilla più sensato (Firefox, Thunderbird e SeaMonkey vanno certamente assieme).
- Common Voice va benissimo ma Mozilla Voice porta allo stesso identico sito web, quindi ho rimosso il secondo.
- Metto a posto i menu in ogni pagina, ordine alfabetico per header e footer, aggiungo Download nell’header.
- Metto a posto `<br>` —> `<br />` e le lettere accentare con HTML code (compatibile con qualsiasi configurazione / browser / browser / ecc.), faccio lo stesso per `<i>` e `<b>` (rispettivamente `<em>` e `<strong>`, sperando di non essermene perse in giro).
- Rimosso riferimento a Firefox Send (EOL).
- Qual è la differenza tra Mozilla e Firefox? doppio nelle FAQ, rimosso.
- Correzioni ortografiche.
- Metto a posto la persona usata nelle pagine (talvolta il “voi”, altre volte il “tu”, standardizzo sul “tu” più informale).

gioxx committed:
Revisione

Ho cercato di dare un ordine ai progetti Mozilla più sensato (Firefox, Thunderbird e SeaMonkey vanno certamente assieme).
Common Voice va benissimo ma Mozilla Voice porta allo stesso identico sito web, quindi ho rimosso il secondo.
Metto a posto i menu in ogni pagina, ordine alfabetico per header e footer, aggiungo Download nell’header.
Metto a posto <br> —> <br /> e le lettere accentare con HTML code (compatibile con qualsiasi configurazione / browser / browser / ecc.).
Rimosso riferimento a Firefox Send (EOL).
Qual è la differenza tra Mozilla e Firefox? doppio nelle FAQ, rimosso.
Correzioni ortografiche.
Metto a posto la persona usata nelle pagine (talvolta il “voi”, altre volte il “tu”, standardizzo sul “tu” più informale).

Sav22999 commented:
@gioxx Per quanto riguardo "Mozilla Voice" lo sostituirei con "DeepSpeech" (prima c'era il sito a sé di "Mozilla Voice", a quanto pare ora è un redirect a Common Voice... Mozilla un giorno crea e il giorno seguente distrugge, boh)

Grazie per le altre correzioni

A proposito del `<br />`, propongo anche di correggere le `<img>` -> `<img />` e i `<meta>`

gioxx committed:
Revisione l10n

Sav22999 committed:
Corretti alcuni typo e aggiungo DeepSpeech

- Aggiunto DeepSpeech nella pagina princiaple (/index.html)
- Corretti alcuni typo
- In "FAQ" ho aggiunto un piccolo richiamo del "Panda Rosso" dal momento che c'è un pulsante "Panda Rosso?"
- Aggiunto pulsante per scaricare il testo "Mozilla Italia" nella sezione "Utilizzo del logo"

gioxx commented:
Visto le tue ulteriori modifiche / suggerimenti adesso, direi che tutto è assolutamente in ordine e più che ragionevole.
Bene così! 😄
